apps/src/dance/DanceAiModal.tsx

```diff
@@ -129,23 +130,30 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
   };
 
   const startAi = async (value: string) => {
-    const response = await doAi(value);
-
-    const params = JSON.parse(response);
-
-    currentAiModalField?.setValue(response);
-
-    const picked = (({
+    const responseJson = await doAi(value);
+    const response = JSON.parse(responseJson);
+    const pickedResponse = (({
```

Comment:
Maybe add a comment explaining this line? Wondering if variable could be renamed...`responseOutputs`? Not sure if that's any better 😄

Reply:
Good call.  I've added a comment.  Will ponder a better name in a future revision!